Gigasecond compilation error on submission

I solved the problem locally and was able to pass all testcases, but during submission through cli my solution fails and I get the following compilation error from the tests tab: 

Results locally: 
```bash
[100%] Built target gigasecond
===============================================================================
All tests passed (5 assertions in 5 test cases)

[100%] Built target test_gigasecond
```

Error from exercism's iteration page on tests tab:
```bash
make[2]: *** No rule to make target '/usr/lib/libboost_date_time.a', needed by 'gigasecond'.  Stop.
make[1]: *** [CMakeFiles/Makefile2:77: CMakeFiles/gigasecond.dir/all] Error 2
make: *** [Makefile:84: all] Error 2
```

--- a/exercises/practice/gigasecond/gigasecond_test.cpp
+++ b/exercises/practice/gigasecond/gigasecond_test.cpp
@@ -9,7 +9,7 @@
 // This problem requires you to install and use the boost date_time library.
 // CMake will try to find and configure it for you if it is installed on your
 // system.
-// See <http://www.boost.org/doc/libs/1_58_0/doc/html/date_time/posix_time.html>
+// See <http://www.boost.org/doc/libs/1_74_0/doc/html/date_time/posix_time.html>
 // for documentation on boost::posix_time
 
 using namespace boost::posix_time;

--- a/exercises/practice/meetup/meetup_test.cpp
+++ b/exercises/practice/meetup/meetup_test.cpp
@@ -9,7 +9,7 @@
 // This problem requires you to install and use the boost date_time library.
 // CMake will try to find and configure it for you if it is installed on your
 // system.
-// See <http://www.boost.org/doc/libs/1_58_0/doc/html/date_time.html> for
+// See <http://www.boost.org/doc/libs/1_74_0/doc/html/date_time.html> for
 // documentation on boost::gregorian::date
 
 TEST_CASE("monteenth_of_May_2013")

